### PR TITLE
103150/combine condition pages

### PR DESCRIPTION
**Pull request by tim-wilde-nimble**
Combines the "Any conditions?" and "Condition details" pages into one and updates the end-of-journey and task list summaries to combine the details (for 'yes'), too.

### CODE REVIEW

File: ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/integration/Record_Decision/103195_create-approved-decision.js [hunk 1]
```diff
@@ -50,11 +49,12 @@ describe('103195 Record new Approved decision', () => {
         cy.projectStateId().should('contain.text', 'APPROVED')
     })
 
+    // Displayed at the end of the journey
     function checkSummary() {
         cy.ApprovedDecisionPreview().should('contain.text', 'APPROVED WITH CONDITIONS')
         cy.ApprovedMadeByPreview().should('contain.text', 'Regional Director for the region')
         cy.AprrovedConditionsSet().should('contain.text', 'Yes')
-        cy.ApprovedConditionDetails().should('contain.text', 'This is a test')
+        cy.AprrovedConditionsSet().should('contain.text', 'This is a test')
```
Comment (jordan-beard) [inline ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/integration/Record_Decision/103195_create-approved-decision.js:57]: This looks like a typo, should be - ApprovedConditionsSet

Comment (jordan-beard) [reply ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/integration/Record_Decision/103195_create-approved-decision.js:57]: Perhaps the command itself is mispelt.

File: ApplyToBecomeInternal/ApplyToBecomeInternal.Tests/Pages/TaskList/Decision/AnyConditionsIntegrationTests.cs [hunk 2]
```diff
@@ -8,93 +9,95 @@
...
+
+			Document.QuerySelector<IHtmlElement>("[href='#ApprovedConditionsSet']")?.Text()
+				.Should().Be("Select whether any conditions were set");
+
+			PageHeading.Should().Be("Were any conditions set?");
+		}
+
+		#region IAsyncLifetime implementation
+
+		public async Task InitializeAsync()
```
Comment (jordan-beard) [inline ApplyToBecomeInternal/ApplyToBecomeInternal.Tests/Pages/TaskList/Decision/AnyConditionsIntegrationTests.cs:84]: Please can we put this at the top, just so its clearer what is happening before each test.

Comment (tim-wilde-nimble) [reply ApplyToBecomeInternal/ApplyToBecomeInternal.Tests/Pages/TaskList/Decision/AnyConditionsIntegrationTests.cs]: Done - was at the bottom as Resharper throws regions down there. I'd put it in a region to keep them together as I _thought_ they'd be separated when sorted during code reformatting - but actually weren't in the end :)

File: ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/integration/Record_Decision/103195_create-approved-decision.js (hunk 1 above)
Comment (tim-wilde-nimble) [reply ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/integration/Record_Decision/103195_create-approved-decision.js:57]: Yes; I'm a bit hesitant to "fix" typos after the last one affected so much. Maybe on a separate task/PR?

**Comment by jordan-beard** [comment]
Morning @tim-wilde-nimble, 
I just pulled your branch to test it out. The back button on the decision date form is no longer working when you select yes.
Also the change button for condition details on the summary page is not working.

**Review by jordan-beard — APPROVED** [review]

**Pull request merged**